packaging/Makefile

```diff
@@ -0,0 +1,19 @@
+.PHONY: skipper eskip docker-push
+
+SKIPPER_VERSION  ?= $(shell git rev-parse HEAD)
```

Comment:
yes, i know. Semantic versioning is in the pipeline, but that comes with some more labor, regarding reviewing the deprecations. Planning to fix the default image version together with that. If this is now not completely unacceptable, then this ugly versions are my shot here. Suggestions?